src/Discord.Net.Rest/Entities/Interactions/InteractionHelper.cs

```diff
@@ -195,6 +200,174 @@ public static async Task DeleteGlobalCommand(BaseDiscordClient client, RestGloba
             await client.ApiClient.DeleteGlobalApplicationCommandAsync(command.Id, options).ConfigureAwait(false);
         }
 
+        public static async Task<RestGlobalUserCommand> CreateGlobalUserCommand(BaseDiscordClient client, Action<UserCommandCreationProperties> func, RequestOptions options = null)
+        {
+            var args = new UserCommandCreationProperties();
+            func(args);
+            return await CreateGlobalUserCommand(client, args, options).ConfigureAwait(false);
+        }
+
+        public static async Task<RestGlobalUserCommand> CreateGlobalUserCommand(BaseDiscordClient client, UserCommandCreationProperties arg, RequestOptions options = null)
```

Comment:
Please simplify by creating a `ApplicationCommandProperties` that can cover both user and message commands, then remove the duplicate methods